lib/ansible/cli/doc.py

```diff
@@ -332,10 +331,10 @@ def tty_ify(cls, text):
         t = cls._CONST.sub("`" + r"\1" + "'", t)        # C(word) => `word'
         t = cls._RULER.sub("\n{0}\n".format("-" * 13), t)   # HORIZONTALLINE => -------
 
-        t = cls._REFTAG.sub(r"", t)  # remove rst :ref:
-        t = cls._TERM.sub(r"", t)  # remove rst :term:
-        t = cls._NOTES.sub(r" Note:", t)  # nicer note
-        t = cls._SEEALSO.sub(r"", t)  # remove seealso
+        # remove rst
+        t = cls._RST_SEEALSO.sub(r"See website for:", t) # seealso is special and need to break
+        t = cls._RST_ADMONITION.sub(r"\1:", t)  # .. note:: to Note: and others
```

Comment:
I wonder if it'd be hard to title-case the matched admonition. They are case insensitive. One can do any of
```rst
.. Note::
.. notE::
.. NOTE::
.. NOTe::
```

Reply:
And with `\1` you preserve what's matched but I'd argue it should always be normalized to `Note:`.

Reply:
r"" does not seem to like title()

Reply:
I think you'd have to extract the value, titlecase it and only then do the replacement.

If I'm reading the docs right, this should work:
```suggestion
        def replace_admonition_title_cased(match_obj):
            return match_obj.group(0).title + ':'
        t = cls._RST_ADMONITION.sub(replace_admonition_title_cased, t)  # .. note:: to Note: and others
```